interfaces/HO-Portal/src/app/program/program-people-affected/program-people-affected.component.ts

```diff
@@ -309,7 +310,7 @@ export class ProgramPeopleAffectedComponent implements OnInit {
     const columnScoreWidth = 90;
     const columnPhoneNumberWidth = 130;
 
-    this.columnsAvailable = [
+    this.standardColumns = [
```

Comment:
I would not use "standard". I think the better, more "programming"/code-related term would be "default" (if that is its purpose).

The term `columnsAvailable` was (I think) to create a collection of ALL possible columns, but that they could be 'turned off or on' based on some setting/value...
That logic hasn't changed right?